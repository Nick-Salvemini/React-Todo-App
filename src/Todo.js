import React from "react";

function Todo({ task, remove }) {
    const containerStyles = {
        display: 'block'
    }

    const tastStyles = {
        display: 'inline'
    }

    return (
        <div style={containerStyles}>
            <p style={tastStyles}>{task}</p>
            <button style={tastStyles} onClick={remove}>X</button>
        </div>
    )
}

export default Todo