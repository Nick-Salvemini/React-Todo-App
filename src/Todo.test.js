import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

describe('Todo component', () => {
    it('renders without crashing', () => {
        render(<Todo task="Sample task" remove={() => { }} />);
    });

    it('matches snapshot', () => {
        const { container } = render(<Todo task="Sample task" remove={() => { }} />);
        expect(container).toMatchSnapshot();
    });
});