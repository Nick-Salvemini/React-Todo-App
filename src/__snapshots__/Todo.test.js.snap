// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo component matches snapshot 1`] = `
<div>
  <div
    style="display: block;"
  >
    <p
      style="display: inline;"
    >
      Sample task
    </p>
    <button
      style="display: inline;"
    >
      X
    </button>
  </div>
</div>
`;
